openfl/interface/aggregator.py

```diff
@@ -52,24 +67,43 @@ def aggregator(context):
     default="plan/cols.yaml",
     type=ClickPath(exists=True),
 )
-def start_(plan, authorized_cols):
-    """Start the aggregator service."""
+@option(
+    "-stg",
```

Comment:
Shorthands are optional, and are recommended for use only if they are common in other command groups (like `-n/--name`, `-k/--key`), or when they easy to infer what they stand for. Let us not add shorthands just because we can.